### DISCUSSION TITLE
Token endpoint getting overwritten when using both Google and Facebook providers together.

### DISCUSSION THREAD

**Discussion by premthebuilder** [nuxt-community/auth-module#1622] [Support Requests]
Hi,
I am building a web app using NuxtJs and I am using nuxtjs/auth-next for OAuth authorization with Google and Facebook auth providers. I have configured post authorization endpoints to fetch token from backend, while it works when I just have one Strategy in nuxt config (either google or facebook), but when I have both then both logins are using the first strategy's endpoint. I have spent a lot of time on this, please share if you have any thoughts on how to resolve this. Thanks!

Here is my auth config in nuxt.config.js

`auth: {
    strategies: {
      facebook: {
        responseType: "code",
        endpoints: {
          token: Config.HOST + "/api/social-login/facebook/",
          userInfo: Config.HOST + "/api/auth/user/"
        },
        clientId: Config.FACEBOOK_CLIENT_ID,
        scope: ["public_profile", "email"],
        refreshToken: {
          property: "refresh_token",
          maxAge: 60 * 60 * 24 * 30
        }
      },
      google: {
        clientId: Config.GOOGLE_CLIENT_ID,
        responseType: "code",
        endpoints: {
          token: Config.HOST + "/api/social-login/google/",
          userInfo: Config.HOST + "/api/auth/user/"
        },
        codeChallengeMethod: "",
        refreshToken: {
          property: "refresh_token",
          maxAge: 60 * 60 * 24 * 30
        }
      }
    }
  }`

**Comment by premthebuilder** [comment]
@pi0 Trying to get your attention to this. Can you please take a look? I have spent days but couldn't figure out why the token endpoint is getting overwritten.

**Comment by ikki-ai** [comment]
Hello,
I struggle a lot with the social login with a server.
did you manage to make it work ?

Thanks a lot
